monkey/monkey_island/cc/environment/set_server_config.py

```diff
@@ -1,7 +1,8 @@
 import argparse
 import json
 import logging
-from pathlib import Path
+
+from monkey_island.cc.environment.environment_config import EnvironmentConfig
```

Comment:
This import won't work since this is an internal script. Use a local import or don't import non-builtins at all